app/views/questionings/_printable.html.erb

```diff
@@ -1,7 +1,8 @@
 <tr class="question <%= "depth-#{qing.depth}" %><%= qing.condition ? ' conditional' : '' %>">
   <td class="name">
     <% if qing.condition %>
-      <div class="condition"><%= t("question.answer_only_caps") %> <%= qing.condition.to_s %></div>
+      <div class="condition"><%= t("question.answer_only_caps") %>
+        <%= qing.decorate.concatenated_conditions %></div>
```

Comment:
Typically you'd indent the child tags on new lines, so:
```
<div>
  <tag1/>
  <tag2/>
</div>
```

unless there is some reason that having whitespace messes things up, but I don't think that's so here.